common/lib/xmodule/xmodule/js/src/capa/display.coffee

```diff
@@ -440,7 +453,7 @@ class @Problem
         @el.addClass 'showed'
         @el.find('.show').attr('disabled', 'disabled')
         @updateProgress response
-        window.SR.readElts(answer_text)
+        window.SR.readText(gettext('The answer to the problem has been shown. Navigate through the problem to read the answer text.'))
```

Comment:
@catong Can you help with the wording of this message?

After "Show Answer" is pressed, answers may be sprinkled throughout the problem (after individual radio buttons/input fields, blocks of text at the end of the problem. etc.). We are moving focus to the top of the problem because there is no single "solution" place to move focus too. 

@cptvitamin suggested that instead of trying to send all "answer text" to the screen reader field (which could result in a LOT of text with little context), we just send a generic message telling the user to review the problem.